arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -39,5 +40,19 @@ public async Task<GetGraphsResponse> GetGraphs()
                 throw await GetApiErrorException(response);
             }
         }
+
+        public async Task<DeleteGraphResponse> DeleteGraphAsync(string graphName, DeleteGraphBody body)
```

Comment:
The documentation indicates that the `dropCollections` is a query parameter, not a JSON body content.